aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -1,6 +1,237 @@
+      pad={{
+        horizontal: size !== 'small' ? 'small' : 'medium',
+        vertical: size !== 'small' ? 'medium' : 'small',
+      }}
+      fill={size !== 'small' ? 'vertical' : 'horizontal'}
+      {...rest}
+    >
+      {size !== 'small' && (
+        <Avatar
+          margin={{ bottom: 'medium' }}
+          src="//s.gravatar.com/avatar/b7fb138d53ba0f573212ccce38a7c43b?s=80"
+        />
+      )}
+      <Box
+        flex="grow"
+        direction={size !== 'small' ? 'column' : 'row'}
+        justify={size === 'small' ? 'between' : undefined}
+      >
+        <Button a11yTitle="Clock" icon={<Clock color="text-xweak" />} />
+        <Button a11yTitle="Apps" icon={<Apps color="text-xweak" />} />
+        <Button a11yTitle="Terminal" icon={<Terminal color="text-xweak" />} />
+      </Box>
+
+      {size !== 'small' && (
+        <Box direction={size !== 'small' ? 'column' : 'row'}>
+          <Button a11yTitle="Chat" icon={<Chat color="text-xweak" />} />
+          <Button
+            a11yTitle="Help"
+            icon={<StatusUnknown color="text-xweak" />}
+          />
+        </Box>
+      )}
+    </Box>
+  );
+};
+
+const ScreenContainer = ({ mobile, ...rest }) => (
+  <ResponsiveContext.Consumer>
+    {size => (
+      <Box
+        background="background-back"
+        width={size === 'small' ? 'medium' : '100%'}
+        height={size === 'small' ? { max: 'large' } : undefined}
+        style={{ position: 'relative' }}
+      >
+        <Box direction="row" fill>
+          {size !== 'small' && <SidebarExample />}
+          <Box
+            overflow="auto"
+            pad={{ horizontal: 'medium', bottom: 'medium' }}
+            flex
+            {...rest}
+          />
+        </Box>
+        {size === 'small' && (
+          <SidebarExample
+            style={{ position: 'absolute', bottom: 0, left: 0 }}
+          />
+        )}
+      </Box>
+    )}
+  </ResponsiveContext.Consumer>
+);
+
+const AppHeaderExample = () => (
+  <Header pad={{ vertical: 'small' }}>
+    <Button>
+      <Box
+        direction="row"
+        align="center"
+        gap="medium"
+        pad={{ vertical: 'small' }}
+        responsive={false}
+      >
+        <Hpe color="plain" />
+        <Box direction="row" gap="xsmall">
+          <Text weight="bold" size="medium">
+            HPE
+          </Text>
+          <Text size="medium">Server</Text>
+        </Box>
+      </Box>
+    </Button>
+    <Box direction="row" gap="small">
+      <Menu
+        label="Master Control"
+        items={[
+          { label: 'Flynn ISO 01' },
+          { label: 'TRON_SERV8r' },
+          { label: 'HPE-Server-101' },
+        ]}
+      />
+    </Box>
+  </Header>
+);
+
+const PageHeaderExample = ({ title }) => (
+  <Header>
+    <Heading margin={{ vertical: 'medium' }} size="small">
+      {title}
+    </Heading>
+    <Button label="Manage" primary />
+  </Header>
+);
 
 export const DashboardExample = ({ ...rest }) => {
```

Comment:
What is the ...rest here -- users won't have access directly to that would they? if we're able to remove it we should.

Reply:
right here yes we dont need rest